core/modules/replica/WorkerSqlRequest.cc

```diff
@@ -259,34 +263,54 @@ database::mysql::Connection::Ptr WorkerSqlRequest::_connector() const {
 }
 
 
-Query WorkerSqlRequest::_query(database::mysql::Connection::Ptr const& conn) const {
+vector<Query> WorkerSqlRequest::_queries(database::mysql::Connection::Ptr const& conn) const {
+
+    vector<Query> queries;
 
     auto const config = serviceProvider()->config();
     auto const workerInfo = config->workerInfo(worker());    
 
+    string const qservChunksTable = conn->sqlId("qservw_worker", "Chunks");
     string const qservDbsTable = conn->sqlId("qservw_worker", "Dbs");
 
     switch (_request.type()) {
 
         case ProtocolRequestSql::QUERY:
-            return Query(_request.query());
+            queries.push_back(Query(_request.query()));
```

Comment:
`emplace_back()` could be slightly more efficient.

Reply:
👍